tools/cbmc/proofs/DNS/DNSgetHostByName_a/Makefile.json

```diff
@@ -0,0 +1,23 @@
+{
+  "ENTRY": "DNSgetHostByName_a",
+  ################################################################
+	# This configuration flag sets callback to 1
+  "callback": 1,
+  "CBMCFLAGS": 
+  [
+  		"--unwind 1",
+  		"--unwindset prvCreateDNSMessage.0:11,prvCreateDNSMessage.1:11,prvGetHostByName.0:11,prvProcessDNSCache.0:5,strlen.0:11,__builtin___strcpy_chk.0:11,strcmp.0:11,xTaskResumeAll.0:11,xTaskResumeAll.1:11",
```

Comment:
I see a lot of `11` here. Is this because it's one more than the length of `pcHostName`?

If so, please define a variable to use both in the Makefile and the C file to avoid duplicating numbers.

```json
"MAX_HOSTNAME_LEN": 10,
"HOSTNAME_UNWIND": "__eval {MAX_HOSTNAME_LEN} + 1",
...
"--unwindset prvCreateDNSMessage.0:{HOSTNAME_UNWIND}",
...
"DEF":
[
  "MAX_HOSTNAME_LEN={MAX_HOSTNAME_LEN}"
]
```
```c
char *pcHostName = malloc(MAX_HOSTNAME_LEN);
```